Vonte orwell fix

Vontens opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Changes to the Orwell station mainly targeting security.
Things changed/added:
- Armory moved close to the Ward office.
- Brigmed moved closer to the entrance of security.
- Warden's office reduced by 1 tile (Sorry Ward).
- Moved and expanded the shooting range so you dont have to fire into the hallways when testing lasers.
- New interrogation room.
- New Security common area with snacks, drinks and vending machines.
- New Crewmonitor avaliable for general secoffs in the middle of the brig.
- New Bomb defusal training area.
- Buffer room for distro in atmos now is the same size as on other stations (9+ tiles).
- Distro now has a gas pipe sensor.
- Science now has a material silo.
- Fixed AI vision in areas that should have them.
- Added an additional door to Atmos.
- Added wall rechargers across the station for potential translators.
- Other small changes (spare sec masks, dominators in lockerooms, APC in CMO office, atmos pipe colors).
## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
The current way the security department is set up in Orwell is very bad for the Wardens due to the long distance between his office, genpop and the armory, with the changes I made this will be amended along with adding features that the department was lacking previously.
## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="6304" height="4256" alt="Orwell-0" src="https://github.com/user-attachments/assets/07682cb7-30c4-4b7b-8b70-ae9896691841" />


## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->
:cl: Vonte
- tweak: reorganised rooms inside Orwell security.
- tweak: other minor changes around the station listed in the description.
- tweak: changes to the engineering locker-room and atmos.
- add: added rooms that the security department in Orwell were missing.

Commit:
Orwell fixes

PubliclyExecutedPig commented:
I have asthetic objections with the new decals, old orwell *was* using the "correct" security decal, also some issues like a camera being in the wall in interrogation or the sentry control panel being behind a drawer. Please fix these issues for approval. Also I'm bothered by
<img width="201" height="114" alt="image" src="https://github.com/user-attachments/assets/07a60a47-4380-48ca-b891-06811a514629" />
normal trims in spots where minitiles should be

PubliclyExecutedPig commented:
Also 
<img width="1088" height="331" alt="image" src="https://github.com/user-attachments/assets/f48e409d-4501-49e1-a527-7c1faea325d4" />
armoury could be extended since its public facing, needs major redecoration, armoury doors are upside down, and plastic tiles are used instead of steel

PubliclyExecutedPig commented:
Please DM me tomorrow so we can work on it, my discord is PubliclyExecutedPig

Vontens commented:
Thanks for the feedback, its my first time trying this stuff and I let some details slip through the cracks it seems.

Commit:
Orwell fixed from fixing

Vontens commented:
I took a final look-over at the map and saw some small piping, camera and light issues. Will be updating the PR with the fix.

Commit:
Orwell final fix trust

Vontens commented:
its nothing big just some leftover pipes from the old rooms that I forgot to remove along with some more lights in the common area. I used the wrong type of camera in some places as well so I fixed that too (also moved a floating camera in the hallway to evac 1 tile west so its actually attached to a wall yw).

Commit:
Orwell final fix forreal this time

Vontens commented:
some more small changes in this one.
added some names to some fax machines that were missing them.
colored the pipes in atmos leading from the gas chambers to clarify what is what.
added an APC to the CMO room since it could be disabled from the outside before.
and lastly changed the Detective room sign to the correct one.

Vontens committed:
Merge branch 'starlight-dev' into Vonte_Orwell_Fix

Commit:
engi fix

Commit:
Merge branch 'starlight-dev' into Vonte_Orwell_Fix

# Conflicts:
#	Resources/Maps/_Starlight/Stations/Orwell.yml

Commit:
Orwell should work

Vontens committed:
Merge branch 'starlight-dev' into Vonte_Orwell_Fix

Vontens commented:
got a bit messy with the merge conflicts but it got all worked out with this I think.

Commit:
door

atmos door

Commit:
Merge branch 'Vonte_Orwell_Fix' of https://github.com/Vontens/space-station-14 into Vonte_Orwell_Fix
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.
